apps/browser/browserservice.cpp

```diff
@@ -26,6 +26,8 @@ const auto ProcDir = QStringLiteral("/proc/%1");
 const auto ErrorPidIsNotPrivileged = QStringLiteral("PID %1 is not in privileged group");
 const auto SailfishBrowserUiService = QStringLiteral("org.sailfishos.browser.ui");
 const auto TransferEngine = QStringLiteral("org.nemo.transferengine");
+const auto Lipstick = QStringLiteral("com.jolla.lipstick");
+const auto DBusErrorMessage = QStringLiteral("PID %1 is not the owner of '%2' or in privileged group");
```

Comment:
Looks good but you could have a bit more descriptive name here. Note that there is another error message a few lines up.

Reply:
Will update to ErrorPidIsNotOwnerServiceOrPrivileged